Livecheckables/kamel.rb

```diff
@@ -0,0 +1,6 @@
+class Kamel
+  livecheck do
+    url :stable
```

Comment:
```suggestion
    url :stable
```

The stable and `head` URLs are both the Git repo in this context but it's probably a _little_ more appropriate to use `:head` here, as the stable URL could change to something different.